accounts/models.py

```diff
@@ -1,19 +1,40 @@
+import os
+from uuid import uuid4
+
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.utils.translation import ugettext_lazy as _
+from django.utils import timezone
+
+from .managers import UserManager
 
-from .managers import CustomUserManager
 
+def create_image_path(instance, filename):
+    # upload_to="%Y/%m/%d" 처럼 날짜로 세분화
+    ymd_path = timezone.now().strftime("%Y/%m/%d")
```

Comment:
이부분 같은 경우엔, 
```suggestion
    ymd_path = timezone.now().strftime("%Y-%m-%d")
```
같은 식으로 `/` 대신 `-` 는 어떨까 싶음. 파일명에 `/` 들어가는게 조금 찜찜하다랄까